### ISSUE TITLE
Update Senna to v3

### ISSUE THREAD

**Issue by adamn** [nltk/nltk#1430]
Senna was upgraded ... 5 years ago.

It seems like the `senna` modules should use v3 instead of v2.  Or maybe this is unused code?  I'm finding it hard to imagine that people are using this and 5 years go by while using the old version.

Granted, people could simply have Senna v3 in the `/usr/share/senna-v2.0` folder.

**Comment by alvations** [comment]
The NLTK API to SENNA should work for `senna-v3.0` too:

``` python
$ wget http://ml.nec-labs.com/senna/senna-v3.0.tgz
$ tar zxvf senna-v3.0.tgz
$ python
>>> from nltk.tag import SennaTagger
>>> tagger = SennaTagger('senna-v3.0')
>>> tagger.tag('What is the airspeed of an unladen swallow ?'.split())
[('What', u'WP'), ('is', u'VBZ'), ('the', u'DT'), ('airspeed', u'NN'), ('of', u'IN'), ('an', u'DT'), ('unladen', u'NN'), ('swallow', u'NN'), ('?', u'.')]
```

As for the doctests, NLTK disregards the version since it downloads and map it to the environmental variable, still it's good to change the docstring so that it shows v3.0 on #1431 =)

**Comment by stevenbird** [comment]
Thanks @adamn, @alvations

**Comment by stevenbird** [comment]
Resolved in #1431
